packages/react/src/components/Lightbox/Lightbox.js

```diff
@@ -0,0 +1,116 @@
+ * @param {string} props.copy Lightbox media short description
+ * @param {string} props.image Lightbox responsive image object
+ * @param {boolean} props.open sets whether the modal is open/close
+ * @param {Function} props.onClose do something on close in addition, return false to completely replace
+ * @returns {*} JSX Object
+ */
+const Lightbox = ({ title, copy, image, open, onClose }) => {
+  /**
+   * Set default image
+   *
+   * @param {*} image responsive image object
+   * @returns {string} default image url
+   */
+  const defaultImage = image => {
+    return image.uri.md
+      ? image.uri.md
+      : image.uri.lg
+      ? image.uri.lg
+      : image.uri.sm;
+  };
+
+  if (!title || !copy || !image) {
+    return null;
+  }
+  return featureFlag(
+    DDS_LIGHTBOX,
+    <section
+      data-autoid={`${stablePrefix}--lightbox`}
+      className={`${prefix}--lightbox`}>
+      <ExpressiveModal open={open} fullwidth={true} onClose={onClose}>
+        <ModalBody>
+          <div className={`${prefix}--lightbox__container`}>
+            <div className={`${prefix}--lightbox__row`}>
+              <div
+                data-autoid={`${stablePrefix}--lightbox__image`}
+                className={`${prefix}--lightbox__image`}>
+                <picture>
+                  {image.uri.lg && (
+                    <source
+                      media="(min-width: 672px)"
+                      srcSet={`${image.uri.lg}`}
+                    />
+                  )}
+                  {image.uri.md && (
+                    <source
+                      media="(min-width: 320px)"
+                      srcSet={`${image.uri.md}`}
+                    />
+                  )}
+                  {image.uri.sm && (
+                    <source
+                      media="(max-width: 320px)"
+                      srcSet={`${image.uri.sm}`}
+                    />
+                  )}
+                  <img src={defaultImage(image)} alt={image.alt}></img>
+                </picture>
+              </div>
+              <div className={`${prefix}--lightbox__content`}>
+                {title && (
+                  <div
+                    data-autoid={`${stablePrefix}--lightbox__content__title`}
+                    className={`${prefix}--lightbox__content__title`}>
+                    {title}
+                  </div>
+                )}
+                {copy && (
+                  <div
+                    data-autoid={`${stablePrefix}--lightbox__content__desc`}
+                    className={`${prefix}--lightbox__content__desc`}>
+                    {copy}
+                  </div>
+                )}
+              </div>
+            </div>
+          </div>
+        </ModalBody>
+      </ExpressiveModal>
+    </section>
+  );
+};
+
+Lightbox.PropTypes = {
+  title: PropTypes.string,
+  copy: PropTypes.string,
+  image: PropTypes.shape({
+    uri: PropTypes.shape,
+    alt: PropTypes.string,
+  }),
```

Comment:
Is it missing the actual shaping of the uri prop here?

Reply:
Right, and are any of these props required? How about default props?

Reply:
Yeah, For alpha phase I , it will work with minimum one url althought we can pass all three sm, md, lg and  we can make lg as required one.

Reply:
updated with [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/882/commits/afbd414592d1948f0861ff4866931e632f6389c6)